src/main/java/com/cgdp/library/dto/author/CreateAuthorRequestDTO.java

```diff
@@ -0,0 +1,6 @@
+package com.cgdp.library.dto.author;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+public record CreateAuthorRequestDTO(@Valid @NotNull String name) {}
```

Comment:
Valid annotation shouldn't be used here also use `@NotBlank` instead
```suggestion
public record CreateAuthorRequestDTO(@NotBlank String name) {}
```